pkg/cli/api_server.go

```diff
@@ -117,7 +118,14 @@ func (c *APIServerCommand) RunUnstarted(ctx context.Context, args []string) (*se
 	// Create basic health check
 	healthcheck.RegisterGRPCHealthCheck(grpcServer)
 
-	p := justification.NewProcessor(kmsClient, c.cfg)
+	validators, pluginClosers, err := plugin.LoadPlugins(c.cfg.PluginDir)
+	if err != nil {
+		logger.Debugw("error loading plugins", "error", err)
```

Comment:
Wouldn't we want to fail and return an error in this case?

Reply:
I was thinking maybe we still want to allow the host to be working by its own if plugins cannot be loaded. But since we are loading from the the plugin directory at deploy time, it makes sense to fail the server so we can manually trim the invalid plugins. Updated the code accordingly